packages/server/src/__tests__/plugin/drainHttpServer/stoppable.test.ts

```diff
@@ -301,12 +302,21 @@ Object.keys(schemes).forEach((schemeName) => {
 
     if (schemeName === 'http') {
       it('with in-flights finishing before grace period ends', async () => {
```

Comment:
This test was causing an open handle / `AggregateError` that I was unfruitful in tracking down. Seems to have been related to spawning a process (maybe the server in the process wasn't being killed?). In any case, I think this change simplifies the test a bit by eliminating the need to spawn a process.